src/cli.ts

```diff
@@ -24,6 +24,11 @@ cli(
 				description: 'Number of messages to generate. (Warning: generating multiple costs more) (default: 1)',
 				alias: 'g',
 			},
+			filename: {
+				type: String,
+				description: 'Excluding a path.',
+				alias: 'f',
```

Comment:
```suggestion
				alias: 'e',
```

Or maybe `x`. Is there a common precedent?